src/animations/README.md

```
@@ -0,0 +1,536 @@
+  });
+  
+  describe('Foam Rendering', () => {
+    test('should determine foam rendering based on threshold', () => {});
+    test('foam should appear on wave crests', () => {});
+  });
+});
+```
+
+## File Naming Conventions
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)